modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/BlogManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Domain.Services;
+
+namespace Volo.CmsKit.Blogs
+{
+    public class BlogManager : DomainService
+    {
+        protected IBlogRepository BlogRepository { get; }
+        
+        public BlogManager(IBlogRepository blogRepository)
+        {
+            BlogRepository = blogRepository;
+        }
+
+        public virtual async Task<Blog> CreateAsync([NotNull] string name, [NotNull] string slug)
```

Comment:
we should also accept the tenantId parameter as optional. The application layer should pass the CurrentTenant.Id.

Reply:
By design, we don't want to move tenantId by code.